test/testcafe/spec/GenericErrorHandling_spec.js

```diff
@@ -2,11 +2,29 @@ import { RequestMock } from 'testcafe';
 import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
 import { checkConsoleMessages } from '../framework/shared';
 import xhr403SecurityAccessDenied from '../../../playground/mocks/data/idp/idx/error-403-security-access-denied';
+import verificationPassword from '../../../playground/mocks/data/idp/idx/authenticator-verification-password';
+import assertionValidationFailure from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-assertion-validation-failure';
+import authdataValidationFailure from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-authdata-validation-failure-during-authentication-challenge';
+import clientdataChallengeMismatch from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-challenge-mismatch';
+import clientdataChallengeMissing from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-challenge-missing';
+import clientdataOriginMismatch from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-origin-mismatch';
+import clientdataOriginMissing from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-origin-missing';
+import clientdataMessageTypeMismatch from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-message-type-mismatch';
+import clientdataParsingFailed from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-clientdata-parsing-failed';
+import invalidEnrollmentRequestData from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-invalid-enrollment-request-data';
+import signatureVerificationFailure from '../../../playground/mocks/data/idp/idx/error-authenticator-webauthn-signature-verification-failure';
+import ChallengePasswordPageObject from '../framework/page-objects/ChallengePasswordPageObject';
 
 const securityAccessDeniedMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(xhr403SecurityAccessDenied, 403);
 
+const challengeFailedResponse = (response) => RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(verificationPassword)
```

Comment:
Make the verification page to password, since the webauthn will always failed in e2e as the following:
![image (50)](https://user-images.githubusercontent.com/47287459/113794046-8d374000-96fe-11eb-8ac4-169de1a5f895.png)
I think it should be fine since we just need to verify the localization here. Please let me know, if it's ok or I need to revert the tests

Reply:
I see that we have unit tests for [ChallengeWebAuthnView](https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js) should we move this e2e test to unit test ? cc: @magizh-okta

Reply:
I think this particular error is a WebAuthn-library-specific error for timeout or when the user clicks cancel (it's part of the set of errors that the browser itself surfaces), and this isn't localized since we're not necessarily sure what error would be returned (it's also opaque on purpose here since the browsers aren't supposed to show time out vs not allowed). The new error messages we show would be localized though.

Reply:
I think this particular error is a WebAuthn-library-specific error for timeout or when the user clicks cancel (it's part of the set of errors that the browser itself surfaces), and this isn't localized since we're not necessarily sure what error would be returned (it's also opaque on purpose here since the browsers aren't supposed to show time out vs not allowed). The new error messages we show would be localized though.

Reply:
If use the unit test, we only test the keys, but not the mocks, if that is fine?

Reply:
I think the best way now is testing in the `i18nTransformer` test with its response mock